velox/docs/functions/spark/aggregate.rst

```diff
@@ -107,10 +107,16 @@ General Aggregate Functions
 
     Returns the sum of `x`.
 
-    Supported types are TINYINT, SMALLINT, INTEGER, BIGINT, REAL and DOUBLE.
+    Supported types are TINYINT, SMALLINT, INTEGER, BIGINT, REAL, DOUBLE and DECIMAL.
 
     When x is of type DOUBLE, the result type is DOUBLE.
     When x is of type REAL, the result type is REAL.
+
+    When x is of type DECIMAL(p, s), the result type is DECIMAL(p + 10, s), and p + 10 will not exceed 38.
+    For DECIMAL type, the initial value of sum is 0. We need to keep sum unchanged if the input is null, as sum
```

Comment:
This comment doesn't seem to belong here. It is not describing user-visible behavior of sum, does it? Sum ignores nulls and starts with 0 for all types, not just decimal. 

non-ASCI mode comment is also not specific to this function. All Spark functions in Velox are matching Spark's non-ANSI mode.

It would be useful to describe what happens on overflow. I assume in case of overflow, sum returns null. Is this true only for decimal or other types? Let's clarify.

Reply:
Only decimal type returns null when calculating overflow. I find this comment in Spark.

> non-decimal type can produce overflowed value under non-ansi mode.

I also conducted a simple test for verification.
```
spark-sql> SELECT sum(col) FROM VALUES (9223372036854775807), (15) AS tab(col);
-9223372036854775794
Time taken: 0.093 seconds, Fetched 1 row(s)
```
I've noticed that the Note section below this doc provides explanations for other types. I will supplement the situation of decimal type overflow there.